Src/DependencyCollector/Shared/Implementation/HttpProcessing.cs

```diff
@@ -249,8 +249,7 @@ internal object OnBegin(object thisObj, bool skipIfNotNew)
         /// <param name="exception">The exception object if any.</param>
         /// <param name="thisObj">This object.</param>                
         /// <param name="returnValue">Return value of the function if any.</param>
-        /// <param name="endTracking">Whether this method should end the tracking or not.</param>
-        internal void OnEnd(object exception, object thisObj, object returnValue, bool endTracking)
+        internal void OnEnd(object exception, object thisObj, object returnValue)
```

Comment:
Callbacks were not introduced in this PR, moving them goes far beyond this pull request,. This PR a critical issue fix and I doubt we should do more refactoring here